Gemfile

```diff
@@ -12,8 +13,10 @@ gem 'uglifier'
 
 group :jekyll_plugins do
   gem 'jekyll-assets'
-  gem 'jekyll_pages_api'
-  gem 'liquid_pluralize'
+  # https://github.com/18F/jekyll_pages_api/pull/21
+  gem 'jekyll_pages_api', git: 'https://github.com/18F/jekyll_pages_api.git', ref: '608affe083e15ef73ac1317e43c7a3f78006630e'
```

Comment:
I just merged 18F/jekyll_pages_api#21. If you cut a new gem, this part can go back to the way it was.